docs/en/ingest-management/fleet/add-fleet-server-on-prem.asciidoc

```diff
@@ -57,6 +67,28 @@ NOTE: This is not required when testing and iterating using the *Quick start* op
 
 // end::cert-prereq[]
 
+[discrete]
+[[default-port-assignments-on-prem]]
+== Default port assignments
+
+//tag::default-port-prereq[]
+
+When {es} or {fleet-server} are deployed, components communicate over well-defined, pre-allocated ports.
+You may need to allow access to these ports. See the following table for default port assignments:
+
+|===
+| Component communication | Default port
+
+| Elastic Agent → {fleet-server} | 8220
+| Elastic Agent → {es} | 9200
+| Elastic Agent → Logstash | 5044
+| Elastic Agent → {fleet} | 5601
+| {fleet-server} → {fleet} | 5601
+| {fleet-server} → {es} | 9200
+|===
+
+// end::default-port-prereq[]
```

Comment:
I like the way you set this up for reuse. 

Questions to help expand my understanding: 
How did you decide which topic should contain the source content?  Is basing out of on prem and reusing in other topics a standard approach?

Reply:
I wish I could give a better response than this, but there was already a tag in the on-prem docs that was acting as source material for cloud, so I just kept with that.